job-service-admin/src/main/java/com/hpe/caf/services/admin/HealthCheck.java

```diff
@@ -0,0 +1,180 @@
+            return;
+        }
+
+        // Attempt to create a open a connection and channel to RabbitMQ. If an error occurs update the
+        // statueResponseMap and return.
+        try {
+            if (!conn.isOpen()) {
+                LOG.error("RabbitMQ Health Check: Unhealthy, unable to open connection");
+                updateStatusResponseWithHealthOfComponent(statusResponseMap, false,
+                        "Attempt to open connection to RabbitMQ failed", "queue");
+                return;
+            } else if (!channel.isOpen()) {
+                LOG.error("RabbitMQ Health Check: Unhealthy, unable to open channel");
+                updateStatusResponseWithHealthOfComponent(statusResponseMap, false,
+                        "Attempt to open channel to RabbitMQ failed", "queue");
+                return;
+            }
+        } catch (final Exception e) {
+            LOG.error("RabbitMQ Health Check: Unhealthy, " + e.toString());
+            updateStatusResponseWithHealthOfComponent(statusResponseMap, false, e.toString(), "queue");
+            return;
+        } finally {
+            conn.close();
+        }
+
+        // There where no issues in attempting to create and open a connection and channel to RabbitMQ.
+        updateStatusResponseWithHealthOfComponent(statusResponseMap, true, null, "queue");
+    }
+
+    private static void updateStatusResponseWithHealthOfComponent(
+            final Map<String, Map<String, String>> statusResponseMap, final boolean isHealthy, final String message,
+            final String component)
+    {
+        final Map<String, String> healthMap = new HashMap<>();
+        if (isHealthy) {
+            healthMap.put("healthy", "true");
+        } else {
+            healthMap.put("healthy", "false");
+        }
+        if (message != null) {
+            healthMap.put("message", message);
+        }
+        statusResponseMap.put(component, healthMap);
+    }
+
+    private static com.rabbitmq.client.Connection createConnection() throws IOException, TimeoutException
+    {
+        final ConnectionOptions lyraOpts = RabbitUtil.createLyraConnectionOptions(System.getenv("CAF_RABBITMQ_HOST"),
+                Integer.parseInt(System.getenv("CAF_RABBITMQ_PORT")),
+                System.getenv("CAF_RABBITMQ_USERNAME"),
+                System.getenv("CAF_RABBITMQ_PASSWORD"));
+        final Config lyraConfig = RabbitUtil.createLyraConfig(1, 30, -1);
+        return RabbitUtil.createRabbitConnection(lyraOpts, lyraConfig);
+    }
+
+    private void performDBHealthCheck(final Map<String, Map<String, String>> statusResponseMap)
+    {
+        Session session = null;
+        try {
+            final SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+            session = sessionFactory.openSession();
+
+            session.doWork(new Work()
+            {
+                public void execute(final Connection connection) throws SQLException
```

Comment:
If you're not going to use a lambda then you should apply the `@Override` annotation.